### PR TITLE
Mkapostolov/exptra android permissions always set

### PR DESCRIPTION
**Pull request by mkapostolov**

### CODE REVIEW

File: troubleshooting/extra-android-permissions-always-set.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: Extra Android permissions always set
```
Comment (ggarabedian) [inline troubleshooting/extra-android-permissions-always-set.md]: Title capitalization and eventually change **set** with **enabled**
For example: **Extra Android Permissions Always Enabled**

File: troubleshooting/extra-android-permissions-always-set.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+title: Extra Android permissions always set
+description: The APK built by the cloud service has more required permissions than I set in my project settings.
+type: troubleshooting
+page_title: Unwanted permissions on Android build
```
Comment (ggarabedian) [inline troubleshooting/extra-android-permissions-always-set.md]: Title capitalization and eventually change the title to:
**Unexpected Permissions in Android Аpp**

File: troubleshooting/extra-android-permissions-always-set.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+title: Extra Android permissions always set
+description: The APK built by the cloud service has more required permissions than I set in my project settings.
```
Comment (ggarabedian) [inline troubleshooting/extra-android-permissions-always-set.md]: I would go for something like this. Personal preference, so feel free to keep yours.
**The built application package (APK) requires more permissions than the specified in my project settings.**

File: troubleshooting/extra-android-permissions-always-set.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    <td>{{site.ac}} app</td>
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Android</td>
+  </tr>
+</table>
+
+## Description
+When I build my application the app asks for permissions that I didn’t set up. Permissions are added although not included in any way in the `config.xml` or the `AndroidManifest.xml`.
```
Comment (ggarabedian) [inline troubleshooting/extra-android-permissions-always-set.md]: Build is a bit out of place here. I would go for something like this:
**When I install my application on a device, it asks for...**

File: troubleshooting/extra-android-permissions-always-set.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+Each plugin injects its required permissions in the `AndroidManifest.xml` file during build time - this is both done for convenience (you do not need to manually enter each required permission by the used plugins) but also to make sure that all required permissions are not omitted or deliberately removed as this would cause the plugin that requires them to not work properly.
+
+## Solution
+1. In order to reduce the number of permissions, disable any plugins that you are not using.
+For example, if the app does not read, delete or add contacts, you should disable the core Contacts plugin. You can do that by following these steps:
+    - Open the app.
+    - From the **Solution Explorer** select **Properties**.
+    - Navigate to the **Plugins** tab.
+    - Locate the **Contacts** plugin.
+    - On the left side, click on the **switch** to disable the plugin.
```
Comment (ggarabedian) [inline troubleshooting/extra-android-permissions-always-set.md]: Somehow I do not believe that this KB should contain information on how to remove plugins. Perhaps you can add a link to the docs with info on how to manage [Core plugins](http://docs.telerik.com/platform/appbuilder/cordova/using-plugins/using-core-plugins/enable-disable-core) and remove this part.

**Review by ggarabedian — CHANGES_REQUESTED** [review]

**Review by ggarabedian — APPROVED** [review]

File: troubleshooting/extra-android-permissions-always-set.md (hunk 1 above)
Comment (mkapostolov) [reply troubleshooting/extra-android-permissions-always-set.md]: Agree. Changed

File: troubleshooting/extra-android-permissions-always-set.md (hunk 3 above)
Comment (mkapostolov) [reply troubleshooting/extra-android-permissions-always-set.md]: Agree. Changed

File: troubleshooting/extra-android-permissions-always-set.md (hunk 2 above)
Comment (mkapostolov) [reply troubleshooting/extra-android-permissions-always-set.md]: Agree. Changed

File: troubleshooting/extra-android-permissions-always-set.md (hunk 4 above)
Comment (mkapostolov) [reply troubleshooting/extra-android-permissions-always-set.md]: Agree. Changed

File: troubleshooting/extra-android-permissions-always-set.md (hunk 5 above)
Comment (mkapostolov) [reply troubleshooting/extra-android-permissions-always-set.md]: Changed

**Comment by AntonDobrev** [comment]
💯

**Pull request merged**